org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/apps/AbstractProcessDialog.java

```diff
@@ -799,9 +822,25 @@ protected void chooseSaveParameter(String saveName, boolean lastRun) {
 		bSave.setEnabled(enabled && !lastRun);
 		bDelete.setEnabled(enabled && fSavedName.getSelectedIndex() > -1
 				&& !lastRun);
-	
+		bShare.setEnabled(enabled);
 	}
-	
+
+	protected void shareSaveParameter(String saveName) {
+		Object o = fSavedName.getSelectedItem(); // TODO create a central method to be used by deleteSaveParameter and shareSaveParameter
+		if (savedParams != null && o != null) {
+			String selected = fSavedName.getSelectedItem().getLabel();
+			for (int i = 0; i < savedParams.size(); i++) {
+				if (savedParams.get(i).getName().equals(selected) && savedParams.get(i).getAD_Client_ID() == Env.getAD_Client_ID(getCtx())) {
+					MPInstance sp = savedParams.get(i);
+					sp.setAD_Org_ID(0);
+					sp.setAD_Role_ID(-1);
+					sp.setAD_User_ID(-1);

```

Comment:
perhaps it would be nice if we can set the sharing scope, i.e for specific user, role or everyone.